<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InformationDialog</class>
 <widget class="QDialog" name="InformationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>106</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>301</width>
    <height>106</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>301</width>
    <height>106</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Device Information</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/active64.png</normaloff>
    <normalon>:/icons/icons/active64.png</normalon>
    <disabledoff>:/icons/icons/greyed64.png</disabledoff>
    <disabledon>:/icons/icons/greyed64.png</disabledon>
    <activeoff>:/icons/icons/active64.png</activeoff>
    <activeon>:/icons/icons/active64.png</activeon>
    <selectedoff>:/icons/icons/selected64.png</selectedoff>
    <selectedon>:/icons/icons/selected64.png</selectedon>:/icons/icons/active64.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>70</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelSerialLabel">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>10</y>
     <width>170</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Serial number:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelSerial">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>116</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelRevisionLabel">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>27</y>
     <width>170</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Hardware revision:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelRevision">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>27</y>
     <width>116</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelMaxPowerLabel">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>44</y>
     <width>170</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Max. power cons. (mA):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelMaxPower">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>44</y>
     <width>116</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InformationDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
